org.wso2.codequalitymatrices/src/main/java/com/wso2/code/quality/metrics/model/Graphql.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.code.quality.metrics.model;
+
+import java.io.Serializable;
+
+/**
+ * This is a bean class used to set the json input structures for calling the github graphql API.
+ *
+ * @since 1.0.0
+ */
+public class Graphql implements Serializable {
+    private String graphqlInputWithHistory;
+    private String graphqlInputWithoutHistory;
+    private static final long serialVersionUID = 42L;
+
+    public Graphql() {
```

Comment:
Without it maven build fails giving this error `com.wso2.code.quality.metrics.model.Graphql is Serializable; consider declaring a serialVersionUID [com.wso2.code.quality.metrics.model.Graphql] At Graphql.java:[lines 32-44] SE_NO_SERIALVERSIONID`. I fixed it as mentioned in the answer by "Jon Skeet" in [http://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it](url) or else we need to avoid implementing `Serializable` interface. What is the best way to handle it?